plugin/node.js

```diff
@@ -98,9 +98,14 @@
     } else if (name in data.modules) {
       result = data.modules[name];
     } else if (data.options.modules && data.options.modules.hasOwnProperty(name)) {
-      var scope = buildWrappingScope(cx.topScope, name);
-      infer.def.load(data.options.modules[name], scope);
-      result = data.modules[name] = scope.exports;
+      var mod = data.options.modules[name];
+      if (typeof(mod) == "string" && mod.charAt(0) == '$') {
+        result = cx.topScope.props[mod.substring(1)];
```

Comment:
That makes good sense. I'll try and get that working. And I'll switch it over to the equal character. Thanks marin.